portfolio.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF">
+  <title>portfolio</title>
+  <link rel="stylesheet" type ="css" href="stylesheets/styles.css">
+</head>
+
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="blog/blog-home.html">blog</a></li>
+        <li><a href="about.html">about me</a></li>
+        <li><a href="portfolio.html" class="current-page">portfolio</a></li>
```

Comment:
I like how you've applied the current-page class so you can style different links based on context. Nice.